README.md

```diff
@@ -48,42 +48,68 @@ Thinking about contributing to RES? Awesome! We just ask that you follow a few s
+  - `lib/css/`: RES css
+  - `lib/environment/`: RES environment code
+  - `lib/images/`: RES images
   - `lib/modules/`: RES modules
-  - `lib/vendor/`: RES vendor libraries
-  - `chrome/`: Chrome-specific RES files
-  - `firefox/`: Firefox-specific RES files
+  - `lib/templates/`: RES templates
+  - `lib/utils/`: RES utilities
+  - `lib/utils/__tests__/`: unit tests
+  - `lib/vendor/`: RES vendor libraries 
+  - `node/`: Node files
   - `safari/`: Safari-specific RES files
-  - `dist/`: build output
-  - `**/__tests__`: unit tests
+  - `utils/`: Misc RES utilities
+  - `CHANGELOG.md`: self-explanatory 
+  - `README.md`: YOU ARE HERE, unless you're browsing on GitHub
+  - `gulpfile.babel.js`: post-build script for zipping folders.
+  - `package.json`: package info, dependencies
+  - `webpack.config.babel.js`: build script
 
 ##### Chrome files
 
-  - `background.js`: the "background page" for RES, necessary for Chrome extensions
-  - `manifest.json`: the project manifest
+  - `images/css-off-small.png`, `images/css-off.png`, `images/css-on-small.png`, `images/css-on.png`: icons for css disable button.
+  - `background.entry.js`: the "background page" for RES, necessary for Chrome extensions
+  - `environment.js`: specific environment settings for Chrome
   - `icon.png`, `icon48.png`, `icon128.png`: icons!
+  - `manifest.json`: the project manifest
+  - `options.html`: options page for chrome extensions
 
+##### Microsoft Edge files
+
+  - `background-edge.html`: the "background page" for RES, necessary for Microsoft Edge extensions
+  - `edge.entry.js`: shim to allow chrome extension code usage
+  - `environment.js`: allows interceptor for Microsoft Edge permissions
+  - `manifest.json`: the project manifest
+  
 ##### Firefox files
 
-  - `index.js`: this is Firefox's sort of "background page" for RES, like what Chrome has, but just a JS file
+  - `images/css-disabled-small.png`, `images/css-disabled.png`, `images/css-off-small.png`, `images/css-off.png`, `images/css-on-small.png`, `images/css-on.png`: icons for css disable button.
+  - `background.entry.js`: the "background page" for RES, necessary for Firefox extensions
+  - `environment.js`: specific environment settings for Firefox
   - `package.json`: the project manifest for the Firefox add-on
 
 ##### Safari files
 
-  - `background-safari.html`: the "background page" for RES, necessary for Safari extensions
   - `Info.plist`: the project manifest
+  - `background-safari.html`: the "background html page" for RES, necessary for Safari extensions
+  - `background.entry.js`: the "background page" for RES, necessary for Safari extensions
+  - `environment.js`: specific environment settings for Safari
   - `icon.png`, `icon48.png`, `icon128.png`: icons!
 
 ## Building development versions of the extension
 
 First time installation:
 
-1. Install [node.js](https://nodejs.org) (version >= 5).
+1. Install [git](https://git-scm.com/).
+1. Install [node.js](https://nodejs.org) (version >= 6).
```

Comment:
Node 5 should still work; at least, [we still test it](https://travis-ci.org/honestbleeps/Reddit-Enhancement-Suite/jobs/135446424#L138)